### PR TITLE
fix msvc warning not enough args for macro

### PR DESCRIPTION
**Pull request by eagleoflqj**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

https://ci.appveyor.com/project/radareorg/radare2/builds/35901972/job/ghq9uefp4vktn6wj?fullLog=true#L1912

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Review by trufae — APPROVED** [review]

File: libr/bin/format/mach0/mach0.c [hunk 1]
```diff
@@ -2465,7 +2465,7 @@ static char *get_name(struct MACH0_(obj_t) *mo, ut32 stridx, bool filter) {
 }
 
 static int walk_exports(struct MACH0_(obj_t) *bin, RExportsIterator iterator, void * ctx) {
-#define ULEB(at) read_uleb128 (&p, end)
+#define ULEB() read_uleb128 (&p, end)
```
Comment (trufae) [inline libr/bin/format/mach0/mach0.c:2468]: Uhm.. is this macro used at all? Because you changed the signature but the build didnt break

Comment (eagleoflqj) [reply libr/bin/format/mach0/mach0.c:2468]: Yes it's used at https://github.com/radareorg/radare2/blob/master/libr/bin/format/mach0/mach0.c#L2509 and afterwards. But what surprised me is the original version was compiled successfully.

Comment (trufae) [reply libr/bin/format/mach0/mach0.c:2468]: interesting. anyway. thanks!

**Comment by XVilka** [comment]
Hiding those arguments is a bad idea, imho. It's better to get rid of this macro whatsoever. Just use `read_uleb128 (&p, end)` directly - it's not that big. Could you please do this instead?

**Comment by ret2libc** [comment]
> Hiding those arguments is a bad idea, imho. It's better to get rid of this macro whatsoever. Just use `read_uleb128 (&p, end)` directly - it's not that big. Could you please do this instead? Same for `SLEB` macro.
> cc @ret2libc

I definitely agree with this. The function to call is not long at all and it is called in just few places.

**Review by XVilka — APPROVED** [review]

**Pull request merged**